doc/man/8/mount.ceph.rst

```diff
@@ -199,6 +199,51 @@ Advanced
     option is enabled, a namespace operation may complete before the MDS
     replies, if it has sufficient capabilities to do so.
 
+:command:`cephx_sign_messages`
+   Enable message signing for msgr1 on-the-wire protocol.  For msgr2.1
+   protocol this option is ignored: message signing is built into 'secure'
+   mode and not offered in 'crc' mode. This is the default.
+
+:command:`nocephx_sign_messages`
+   Disable message signing for msgr1 on-the-wire protocol. For msgr2.1
+   protocol this option is ignored.
```

Comment:
`cephx_sign_messages` and `nocephx_sign_messages` are obscure options for a legacy protocol.  We probably don't need to duplicate their descriptions here.

Reply:
Sure, will remove it.